gui/drawer.go

```diff
@@ -125,17 +126,22 @@ L:
 	d.chartDrawing.Store(false)
 }
 
-func (d *drawer) redrawGauge(ctx context.Context, maxSize int) {
-	var count float64
-	size := float64(maxSize)
+func (d *drawer) redrawGauge(ctx context.Context, duration time.Duration) {
+	totalTime := float64(duration)
```

Comment:
I'd say we don't have to do redraw the gauge as soon as a result arrives anymore.

```suggestion
	ticker := time.NewTicker(redrawInterval)
	defer ticker.Stop()

	totalTime := float64(duration)
```